mantidproject/mantid
# DynamicPDF: documentation for the SliceSelector widget

I added some pretty (hopefully useful) pictures of the Slice Selector and some descriptive text
**To test:**
- Open Mantidplot.
- Interfaces --> Dynamic PDF --> Dynamic PDF Bacground Remover.
- Click in the `?` button.
- Browser to the `Slice Selector` section and make sure description makes sense.
- That's all! :dancers:



Fixes #16715.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
